### PR TITLE
[patch] Fix authorize

### PR DESCRIPTION
**Pull request by ssunorz**
# Description

_description_with_details_and_reasoning_

## Type of change

- [x] Bug fix
- [ ] New feature
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Non-code changes (update documentation, pipeline, etc.)

### Flags

- [ ] breaks backward compatibility
- [ ] requires a documentation update
- [ ] has untestable code

## Related issue/PR

**Delete if not applicable**
- Fixes #_issue_
- Closes #_PR_

---

## Checklist

- [ ] Followed the guidelines in the CONTRIBUTING document
- [ ] Added prefix `[major]`/`[minor]`/`[patch]`/`[skip]` in the PR title
- [ ] Tested and linted the code
- [ ] Commented the code
- [ ] Made corresponding changes to the documentation
- [ ] Confirmed no dropping in test coverage (by [Codecov](https://codecov.io/gh/yahoojapan/athenz-authorizer/pulls))
- [ ] Passed all pipeline checking
- [ ] Approved by >1 reviewer

## Checklist for maintainer
- [ ] Use `Squash and merge`
- [ ] Double-confirm the merge message has prefix `[major]`/`[minor]`/`[patch]`/`[skip]`
- [ ] Delete the branch after merge

### CODE REVIEW

File: authorizerd_test.go [hunk 1]
```diff
@@ -927,18 +928,110 @@ func Test_authorizer_verify(t *testing.T) {
...
+				wantResult: p,
+				checkFunc: func(prov *authority) error {
+					_, ok := prov.cache.Get("dummyTok")
+					if !ok {
+						return errors.New("cannot get dummyTok from cache")
+					}
+					return nil
+				},
+			}
+		}(),
```
Comment (kyfujisa) [inline authorizerd_test.go:1030]: How about adding a test case like the following?
- disableA: true
- wantResult: p
- checkFunc: check `prov.cache.Get("dummyTok" + "dummyAct" + "dummyRes")` is ok or not

Comment (ssunorz) [reply authorizerd_test.go:1030]: Fixed
https://github.com/yahoojapan/athenz-authorizer/pull/74/commits/734d5b538145de352454bb968c8a672635f69e5f

Comment (kyfujisa) [reply authorizerd_test.go:1030]: Sorry, I made a mistake.
What I meant to say was the test with disablePolicyd set to false.

Fixed the comment above.

Comment (ssunorz) [reply authorizerd_test.go:1030]: Fixed
https://github.com/yahoojapan/athenz-authorizer/pull/74/commits/f8f3c1259d2eb446f10255846e6c2162f84b2583

**Pull request merged**